src/components/AssetsTable/index.jsx

```diff
@@ -197,6 +196,38 @@ export class AssetsTable extends React.Component {
     };
   }
 
+  getUploadStatusFields(assetsStatus) {
+    switch (assetsStatus.type) {
+      case assetActions.UPLOAD_ASSET_SUCCESS:
+        return {
+          alertType: 'success',
+          alertDialog: `${assetsStatus.loadedCount} files successfully uploaded.`,
```

Comment:
So, I like the idea of having a total count. I'm just not sure how to handle the edge cases around it though, e.g.
* Some number of files were uploaded, then another set of files were uploaded. Maybe we can simply keep an aggregated count of files uploaded?
* If the status alert/message is closed, the state is in turn cleared out. We'd need to include logic to persist the various counts.

Maybe what we need is better text? I'll check in w/ docs and report back.

Reply:
@srpearce -- I've updated the status message when files are successfully uploaded. This happens incrementally as each file is uploaded. I've attached a screenshot of the message.

The issue is that we don't keep track of the total number of files being uploaded, so it might be that 2 out of the 3 files that you've uploaded have completed and you see something like the attached screenshot. In lieu of having a total count of files being uploaded, is there any creative copy updates that we could do to indicate that files have been updated and that there may or may not be more in progress? Is that asking too much?!?

![screen shot 2018-01-04 at 10 39 30 am](https://user-images.githubusercontent.com/5265058/34571065-9543b010-f13b-11e7-9fc4-cd7a2c49a8ec.png)

Reply:
@dsjen I can't think of a good way to write around this, unfortunately. The only options I can see are:
* Don't update incrementally—only add a message when the file uploader is finished with everything (definitely not ideal)
* Change the period after "n files successfully uploaded" to an ellipsis if any other files are still uploading, and back to a period when all files are done (also not ideal because it's probably way too subtle for anyone to notice)

How hard would it be to keep track of an aggregated number of files and go with Tyler's suggestion ("n of x files uploaded")? I think that's how most UIs I've seen do it.